apps/emqx_bridge_gcp_pubsub/src/emqx_bridge_gcp_pubsub_producer_schema.erl

```diff
@@ -101,14 +103,16 @@ fields(Field) when
 
 connector_config_fields() ->
     emqx_bridge_gcp_pubsub:fields(connector_config) ++
-        emqx_resource_schema:fields("resource_opts").
+        emqx_connector_schema:resource_opts_ref(?MODULE, connector_resource_opts).
```

Comment:
This change is not compatible right?
Just a note: Given that v2 schema is not yet implemented from dashboard, it's ok to break compatibility.

Reply:
Correct, it'll reduce the number of accepted fields here.  I've added a breaking change changelog for it.